sonar-csharp-plugin/src/main/java/org/sonar/plugins/csharp/CSharpPlugin.java

```diff
@@ -60,6 +60,7 @@ public void define(Context context) {
       DotNetSensor.class,
       CSharpConfiguration.class,
       CSharpGlobalProtobufFileProcessor.class,
+      CSharpSolutionConfiguration.class,
```

Comment:
What's the difference between this class and the existing `CSharpConfiguration.class`?
Are the settings for the new class per-SQ-project, whereas the settings for the `CSharpConfiguration.class` are per-SQ-module?

Reply:
@duncanp-sonar yes, that's the idea.